### ISSUE TITLE
Failing test cases through maven

### ISSUE THREAD

**Issue by ayushsingh01042003** [ERDDAP/erddap#140]
#### We have 3 errors on running the test cases through maven.

**To Reproduce**
1. After we get a "BUILD SUCCESS" message on running `mvn compile`
2. Run the command `mvn package` in _webapps/erddap_

This should result in the following error message
```
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   HashDigestTests.basicTest:21 » Runtime 
ERROR in Test.ensureEqual(Strings) line #1, col #1 '3[end]'!='9[end]':
Specifically, at line #1, col #1:
s1: 327fbb2aa6c6297d4fdd5fdf4b14e289[end]
s2: 96e9b4d974b734874a66f433b276a41a[end]
    ^

[ERROR]   NcHelperTests.testUnlimited:578 » FileNotFound /temp/unlimited.nc (No such file or directory)
[ERROR]   TableTests.testReadOpendapSequenceSpeed:2715 Simple readOpendapSequence took too long (time=2461ms).
[INFO] 
[ERROR] Tests run: 57, Failures: 0, Errors: 3, Skipped: 0
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:01 min
[INFO] Finished at: 2024-03-25T20:46:00+05:30
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.2.1:test (default-test) on project ERDDAP: 
[ERROR] 
[ERROR] Please refer to /usr/local/Tomcat/webapps/erddap/target/surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.

```

**Comment by ChrisJohnNOAA** [comment]
I wonder if the license.txt hash issue is related to git behavior on different systems (it can convert line endings on checkout and on commit automatically to make files match what is expected by your OS). This would lead to Linux machines potentially having a different hash for a txt file than a Windows system.

For the NcHelperTests.testUnlimited error, that seems to be a hardcoded path I missed updating previously. I'm fixing that as part of my next update to the maven tests.

For TableTests.testReadOpendapSequenceSpeed, there's no real error there. There probably should not be asserts for time taken because what's good on one machine may be slow on another. Additionally, with the change to using JUnit for testing (instead of the old approach), tests can now be run in parallel. The time based asserts were often set assuming there were no other significant stresses on the system (like other tests).

**Comment by ChrisJohnNOAA** [comment]
I believe this has been fixed with the merge of #142
